app/controllers/topics_controller.rb

```diff
@@ -66,7 +64,7 @@ def relative_numbers
 
     topicsAlreadyHas = params[:network] ? params[:network].split(',').map(&:to_i) : []
 
-    @alltopics = policy_scope(Topic.relatives1(@topic.id)).to_a.concat(policy_scope(Topic.relatives2(@topic.id)).to_a).uniq
```

Comment:
lines 69 and 91 used to have .uniq, but now that's inside the `relatives` scope.